index.html

```diff
@@ -1,27 +1,104 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
+  <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title></title>
-    <link rel=“stylesheet” href=“https://use.fontawesome.com/releases/v5.5.0/css/all.css”>
+    <title>Apple clone</title>
     <link
-        href="https://fonts.googleapis.com/css2?family=Montserrat:wght@900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
-        rel="stylesheet" />
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,200;0,300;0,400;0,500;0,600;1,500;1,600&display=swap"
+      rel="stylesheet"
+    />
     <link
-        href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,600;0,700;1,200;1,300;1,400;1,600;1,700&display=swap"
-        rel="stylesheet" />
-
-    <link href="favicon.icon" rel="icon" type="image/x-icon" />
+      href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,600;0,700;1,200;1,300;1,400;1,600;1,700&display=swap"
+      rel="stylesheet"
+    />
     <script src="https://kit.fontawesome.com/a076d05399.js"></script>
-
     <link rel="stylesheet" href="styles/style.css" />
-</head>
+  </head>
 
-<body>
-    <div>
+  <body>
+    <main class="grid-container">
+      <section class="main-section">
+        <div class="container">
+          <nav>
+            <ul>
+              <li>
+                <a href="#"><i class="fab fa-apple fa-2x"></i></a>
+              </li>
+              <li><a href="#">Store</a></li>
+              <li><a href="#">Mac</a></li>
+              <li><a href="#">iPod</a></li>
+              <li><a href="#">iPhone</a></li>
+              <li><a href="#">iPad</a></li>
+              <li><a href="#">iTunes</a></li>
+              <li><a href="#">Support</a></li>
+              <li>
+                <i class="fas fa-search fa-sm"></i
+                ><input type="search" name="search" />
+              </li>
+            </ul>
+          </nav>
+          <h1>What will your verse be?</h1>
+          <h2>
+            Everyone has something to share. See how people are using <br />
+            iPad in their unique ways to add to the world’s story.
+          </h2>
+          <a href="#"
```

Comment:
- [ ] [OPTIONAL] On hover, this link should have a white line underneath. [See original page](https://web.archive.org/web/20140301004610/http://www.apple.com/)

- [ ] [OPTIONAL] Reduce the font-weight of text in this section to make them 'thinner'. [See original page](https://web.archive.org/web/20140301004610/http://www.apple.com/)